crates/storage/errors/Cargo.toml

```diff
@@ -15,8 +15,9 @@ alloy-rlp.workspace = true
 reth-primitives.workspace = true
 reth-fs-util.workspace = true
 
-thiserror-no-std = { workspace = true, default-features = false }
+thiserror-no-std = { workspace = true, default-features = false, optional = true}
```

Comment:
I believe we can actually remove this now

Reply:
done